src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,40 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.command.Command;
+import duke.exception.DukeException;
+import duke.task.TaskList;
+
+/**
+ * Duke class which is the main entry point to run the application.
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
```

Comment:
tasks might be a misleading name for a TaskList object, perhaps a more specific name would be better?

Reply:
Sure, thanks!